### ISSUE TITLE
array type declaration

### ISSUE THREAD

**Issue by nifa-dev** [rectorphp/rector#826]
First time user, so I'm sure this is an error on my end that I'm struggling with figuring out.  

I'm using rector to upgrade from CakePHP 3.3, ultimately to 3.7.  I've run into an issue where it's looking for the 'array' class.  I'm getting this error:

```
In Broker.php line 242:
                                                                                                           
  [PHPStan\Broker\ClassNotFoundException]                                                                  
  Class array was not found while trying to analyse it - autoloading is probably not configured properly.
```

Here is my code, simple just for an example:

```php
<?php

class test {
    public function init( array $data) {
        return $data;
    }
}
```

I'm running the following command `vendor/bin/rector process src/Model/Table/test.php --level cakephp34 --dry-run -vvv`

Where have I run aground with this array issue?  Or am I doing it right and there's something else wrong?

**Comment by TomasVotruba** [comment]
Hi, thanks for report with deatiled example and command you've run.

I'll look into it.

**Comment by TomasVotruba** [comment]
Fixed and released in v0.3.24
